GitCommands/Git/AheadBehindData.cs

```diff
@@ -2,15 +2,21 @@
 {
     public struct AheadBehindData
     {
+        public static readonly string Gone = "gone";
```

Comment:
Make translatable or replace with a character, e.g. a dash.

Reply:
(What I was most unsure about.)
"gone" is what the Git command writes, but I prefer "-" myself too.
(The change is on line 13, the const is still needed.)

Reply:
> I was going to ask what git uses, but you've already said. If "gone" is
git's lingo I'm ok with it. Though my first reaction was to have "deleted"
or "removed".
"-" to me is akin "n/a", which is probably not really suitable in here. I
think this may be appropriate for "not pushed" case....

@RussKie 
Changing to "-" right now
Should we have this translatable if sticking with "gone"?

For branches that have lost the tracking (second part of #8553), should we reserve something?
(I want to display this in FormPush at least)

Reply:
If it's a git's string I think we can have it as is. Need to have a comment in the code why it is not translated though.

Reply:
for-each-ref is a plumbing command and should not be translated, otherwise also ahead/behind could be incorrect.
I added one more check (made the parsing even more complicated) to handle garbage, now it could be interpreted as "0" changes.

I checked the Git code and 'gone' as well as 'ahead'/'behind' can be translated in other printouts. I tried to get command cli to be presented in something else than en, but that failed (Just "LANGUAGE=de_DE git status" should be enough). Maybe not included with git-for-windows? So maybe it should be translated later. But the translated "expression" can be harder to understand, git gone yields some Google hits.

Not sure if the comment is good though...